src/com/company/Main.java

```diff
@@ -3,6 +3,38 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+	// Задача 1
+        System.out.println("Задача 1");
+        int i=1;
+        while (i<=10) {
+            System.out.print(i+" ");
+            i++;
+        }
+        i--;
+        System.out.println();
+        for(;i>0;i--) {
+            System.out.print(i+" ");
+        }
+        System.out.println();
+        System.out.println();
+
+        //Задача 2
+        System.out.println("Задача 2");
+        int firstFridayNumber=7;
+        for(int i2=firstFridayNumber;i2<=31;i2+=7) {
```

Comment:
Можете использовать ctrl+alt+L для автоформатирования
```suggestion
        for(int i2 = firstFridayNumber; i2 <= 31; i2 += 7) {
```